atom/browser/api/atom_api_web_contents.cc

```diff
@@ -971,6 +989,50 @@ void WebContents::AuthorizePlugin(mate::Arguments* args) {
       web_contents(), true, resource_id);
 }
 
+void WebContents::AddWord(mate::Arguments* args) {
```

Comment:
will do. and this will helps for migrating old words in legacy custom dictionary.